src/main/java/com/moodybluez/enterprise/service/MoodServiceStub.java

```diff
@@ -1,15 +0,0 @@
-package com.moodybluez.enterprise.service;
-
-import com.moodybluez.enterprise.dto.Mood;
-import org.springframework.stereotype.Component;
-
-@Component
-public class MoodServiceStub implements IMoodService {
-    @Override
-    public Mood fetchById(int id) {
-        Mood mood = new Mood();
-        mood.setMoodID(3);
-        mood.setMoodDesc("Sad");
-        return mood;
-    }
-}
```

Comment:
Removed these files, because the logic here should really be in the DAO layer, which I have created and that includes necessary methods created in these service layer files.